src/qt/sendcoinsdialog.cpp

```diff
@@ -509,7 +509,7 @@ void SendCoinsDialog::sendButtonClicked([[maybe_unused]] bool checked)
         // Fill without signing
         const auto err{model->wallet().fillPSBT(std::nullopt, /*sign=*/false, /*bip32derivs=*/true, /*n_signed=*/nullptr, psbtx, complete)};
```

Comment:
```suggestion
        const auto result{model->wallet().fillPSBT(std::nullopt, /*sign=*/false, /*bip32derivs=*/true, /*n_signed=*/nullptr, psbtx, complete)};
```
Should this be defined as `result` too, as you did in other parts of the code?

Reply:
Thanks, updated in 4e038c4b7a6fbfaafe1f2570592e2bb751f9be09